pkg/metricscollector/prommetrics.go

```diff
@@ -35,16 +36,16 @@ var (
 		prometheus.GaugeOpts{
 			Namespace: DefaultPromMetricsNamespace,
 			Name:      "build_info",
-			Help:      "A metric with a constant '1' value labeled by version, git_commit and goversion from which KEDA was built.",
+			Help:      "Info metric, with static information about KEDA build like: version, git commit and Golang runtime info.",
 		},
 		[]string{"version", "git_commit", "goversion", "goos", "goarch"},
 	)
-	scalerErrorsTotal = prometheus.NewCounterVec(
+	scalerErrorsTotalDeprecated = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: DefaultPromMetricsNamespace,
 			Subsystem: "scaler",
 			Name:      "errors_total",
-			Help:      "Total number of errors for all scalers",
+			Help:      "DEPRECATED - use a `sum(scaler_errors_total{scaler!=\"\"})` over all scalers",
```

Comment:
Based on our deprecation policy, we have to keep deprecated things 2 versions, so as the next version is 2.13 (when this will be released), these metrics will be removed as part of 2.15. Could you add a note on the description with the removal version? (on every deprecated metric)

Reply:
annotated these, as an example: 

```
			Help:      "DEPRECATED - will be removed in 2.15 - Use `scaled_object_validation_errors_total` instead.",
```